server/views/match/search.njk

```diff
@@ -2,146 +2,173 @@
-				</dd>
-			</div>
-			<div class="search-identity-table__table-cell">
-				<dt>
-					<strong>Application</strong>
-				</dt>
-				<dd>
-					<a href="{{applicationPath}}">View <span class="govuk-visually-hidden">application</span></a>
-				</dd>
-			</div>
-
-		</div>
-	</dl>
-	<hr/>
-
-	<h2 class="govuk-heading-m">{{bedSearchResults.resultsBedCount}} matching beds in {{bedSearchResults.resultsRoomCount}} rooms in {{bedSearchResults.resultsPremisesCount}} premises</h2>
-	<form action="{{formPath}}" method="post">
-		<div class="bed-search-inputs">
-			<input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
-			<input type="hidden" name="crn" value="{{crn}}"/>
-			<input type="hidden" name="startDate" value="{{startDate}}"/>
-			<input type="hidden" name="applicationId" value="{{applicationId}}"/>
-			<input type="hidden" name="assessmentId" value="{{assessmentId}}"/>
-
-			{{
-				govukInput({
-				id: "durationDays",
-				name: "durationDays",
-				value: durationDays,
-				type: "number",
-				label: {
-				text: "Length of stay"
-				},
-				classes: "govuk-input--width-2",
-				suffix: {
-					text: "weeks"
-				}
-				})
-			}}
-
-			{{ govukDateInput({
-				id: "startDate",
-				namePrefix: "startDate",
-				fieldset: {
-				legend: {
-					text: "Expected arrival",
-					classes: "govuk-fieldset__legend--s"
-					}
-				},
-				items: dateFieldValues('startDate', errors),
-				errorMessage: errors.startDate
-			}) }}
-
-			{{ govukInput({
-				id: "postcodeDistrict",
-				name: "postcodeDistrict",
-				value: postcodeDistrict,
-				label: {
-				text: "Preferred location"
-				},
-				classes: "govuk-input--width-3"
-     		}) }}
-
-			{{ govukInput({
-				id: "maxDistanceMiles",
-				name: "maxDistanceMiles",
-				value: maxDistanceMiles,
-				label: {
-				text: "Radius"
-				},
-				classes: "govuk-input--width-3", 
-				suffix: {
-					text: "miles"
-				}
-        	}) }}
-
-			<div class="button">
-				{{ govukButton({
-          		text: "Update"
-        	}) }}
+	</div>
+	<div class="govuk-grid-row">
+		<div class="govuk-width-container">
+
+			<form action="{{formPath}}" method="post">
+				<h2 class="govuk-heading-m">{{bedSearchResults.resultsBedCount}} matching beds in {{bedSearchResults.resultsRoomCount}} rooms in {{bedSearchResults.resultsPremisesCount}} premises</h2>
+
+				<input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+				<input type="hidden" name="crn" value="{{crn}}"/>
+				<input type="hidden" name="startDate" value="{{startDate}}"/>
+				<input type="hidden" name="applicationId" value="{{applicationId}}"/>
+				<input type="hidden" name="assessmentId" value="{{assessmentId}}"/>
+				{% for requiredCharacteristic in requiredCharacteristics %}
+					<input type="hidden" name="requiredCharacteristics[]" value="{{requiredCharacteristic}}"/>
+					{%endfor%}
+
+					<div class="govuk-grid-column-one-third">
+						{{govukCheckboxes({
```

Comment:
Could we use [Smaller Checkboxes](https://design-system.service.gov.uk/components/checkboxes/#smaller-checkboxes) here?